content/es/docs/concepts/architecture/control-plane-node-communication.md

```diff
@@ -0,0 +1,100 @@
+---
+reviewers:
+- dchen1107
+- liggitt
```

Comment:
Remove the reviewer's section as those users are not reviewing this localization itself.

```suggestion
```